Sources/SwiftProtobuf/BinaryDecodingOptions.swift

```diff
@@ -21,5 +21,18 @@ public struct BinaryDecodingOptions {
   /// while parsing.
   public var messageDepthLimit: Int = 100
 
+  /// Discard unknown fields while parsing.
+  ///
+  /// The Protobuf binary format allows unknown fields to be still parsed
+  /// so the schema can be expanded without requiring all readers to be updated.
+  /// This works in part by haivng any unknown fields preserved so they can
+  /// be relayed on without loss. For a while the proto3 syntax definition
+  /// called for unknown fields to be dropped, but that lead to problems in
+  /// some case. The default is to follow the spec and keep them, but setting
+  /// this option to `true` allows a developer to strip them during a parse
+  /// incase they have a specific need to drop the unknown fields from the
```

Comment:
We should probably include a sentence specifying the default as part of the doc comment.  In this case,
```
By default, the parser does not discard unknown fields.
```